sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -4,28 +4,100 @@
     :records-count="offeringsAndCount.count"
   >
     <template #actions>
-      <v-row class="m-0 p-0">
-        <v-text-field
-          density="compact"
-          label="Search Offering Name"
-          variant="outlined"
-          v-model="searchBox"
-          data-cy="searchBox"
-          @keyup.enter="searchOfferingTable"
-          prepend-inner-icon="mdi-magnify"
-          hide-details="auto"
-        />
-        <v-btn
-          v-if="allowOfferingEdit"
-          class="ml-2 float-right"
-          @click="goToAddNewOffering()"
-          color="primary"
-          prepend-icon="fa:fa fa-plus-circle"
-          data-cy="addNewOfferingButton"
-        >
-          Add offering
-        </v-btn>
-      </v-row>
+      <v-form ref="searchOfferingsForm">
+        <div class="d-flex flex-wrap align-center ga-3">
+          <!-- Search Field -->
+          <v-text-field
+            density="compact"
+            label="Search Offering Name"
+            variant="outlined"
+            v-model="searchBox"
+            data-cy="searchBox"
+            @keyup.enter="searchOfferingTable"
+            prepend-inner-icon="mdi-magnify"
+            hide-details="auto"
+            class="flex-grow-1"
+          />
+
+          <!-- Date Range Group - stays together -->
+          <div class="d-flex align-center ga-2 flex-grow-1">
+            <v-date-input
+              density="compact"
+              variant="outlined"
+              label="From (Study Start Date)"
+              input-format="yyyy-mm-dd"
```

Comment:
Please use the existing const. 

https://github.com/bcgov/SIMS/blob/ed3f9c5a180c5b45bac0ee91c161ae0a63e23251/sources/packages/web/src/composables/useFormatters.ts#L23